### ISSUE TITLE
Decide whether Lighthouse references should repeat all resources in "Resources" section

### ISSUE THREAD

**Issue by kaycebasques** [GoogleChrome/web.dev#1480]
@mfriesenhahn and I are kicking around the idea of listing every link that is mentioned in a Lighthouse reference in the **Resources** section at the bottom. So for example in the screenshot below we'd add the `third-party scripts can significantly affect your page load performance` link, `main thread` link, and so on to the **Resources** section.

On first impression it seems kinda nice, but after trying to implement it, it feels like a lot of maintenance overhead for unclear benefit. So this thread is to decide whether we want to do this across the board. If we do we can update this issue to update all of the Lighthouse reference docs in one fell swoop.

If we don't do it, we should decide what the **Resources** section is for. My suggestion is that we use it to mention other relevant resources that haven't already been mentioned in the doc. Either way, we should also clarify somewhere in the handbook the purpose of the **Resources** section.

![localhost_8080_third-party-summary_](https://user-images.githubusercontent.com/4713486/64399374-15ce9580-d01d-11e9-88af-4956e807ec9e.png)

**Comment by mfriesenhahn** [comment]
A little background:

   - When I started tidying the audit docs, they had a "More information"
   section at the bottom that always included a link to the audit source code,
   sometimes included links from the post body, and sometimes included links
   to resources *not *mentioned in the post. So, I was looking for a
   principle about what to put in that section.
   - I initially tried changing it to "More resources" and only including
   links not already in the post. However, I kept bumping into cases where it
   seemed helpful to include links from the post. (For example, to make
   sentences flow well, link text in the body often describes the relevant
   concept rather than providing the name of the resource.) It seemed like the
   predictability of having a collection of essential links might be helpful
   so readers (especially return readers) don't need to go hunting through the
   post to find the resource they need.
   - To keep the link lists from getting *too *long, I was thinking of only
   including the links from the post that are immediately relevant to the
   audit. But that does require some judgement, so there may be a bit of
   variation across authors. If that's a concern, or if folks think a
   predictable resource list doesn't provide value, as Kayce notes, we
   shouldn't take this tack.


*Michael Friesenhahn*
Technical Writer (provided by Genesis10)
mfriesenhahn@google.com
